src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -30,7 +32,9 @@
         PRIVILEGES_COOWNER.put(Const.InstructorPermissions.CAN_SUBMIT_SESSION_IN_SECTIONS, true);
         PRIVILEGES_COOWNER.put(Const.InstructorPermissions.CAN_MODIFY_SESSION_COMMENT_IN_SECTIONS, true);
 
-        PRIVILEGES_MANAGER.put(Const.InstructorPermissions.CAN_MODIFY_COURSE, false);
+        PRIVILEGES_MANAGER.put(Const.InstructorPermissions.CAN_EDIT_COURSE, false);
```

Comment:
Editing the course should be true for a manager. They just can't delete/restore courses as per the role prompt on the frontend